Binary files a/src/.os_unix.c.swp and b/src/.os_unix.c.swp differ
diff -rNU 8 a/src/os_unix.c b/src/os_unix.c
--- a/src/os_unix.c	2013-09-06 09:55:01.000000000 -0500
+++ b/src/os_unix.c	2013-10-27 23:19:49.000000000 -0500
@@ -799,16 +799,20 @@
 #if defined(HAVE_SIGALTSTACK) || defined(HAVE_SIGSTACK)
 /*
  * Support for using the signal stack.
  * This helps when we run out of stack space, which causes a SIGSEGV.  The
  * signal handler then must run on another stack, since the normal stack is
  * completely full.
  */
 
+#if defined(__APPLE__)
+#include <AvailabilityMacros.h>
+#endif
+
 #ifndef SIGSTKSZ
 # define SIGSTKSZ 8000    /* just a guess of how much stack is needed... */
 #endif
 
 # ifdef HAVE_SIGALTSTACK
 static stack_t sigstk;			/* for sigaltstack() */
 # else
 static struct sigstack sigstk;		/* for sigstack() */
